src/navigation/actions.js

```diff
@@ -254,9 +254,12 @@ export const createSupplierInvoice = (otherParty, currentUser) => dispatch => {
  * @param {Object} stocktake realm Stocktake object
  * @param {Array}  itemIds   Array of item id strings that should be the new
  *                           Items in the stocktake.
+ *  * @param {String} name   Name of the stocktake for updating. Cannot update to remove completely.
```

Comment:
Is the indent and extra `*` intentional?

Reply:
special one 
82fc96e